docusaurus/docs/e2e-test-a-plugin/getting-started.md

```diff
@@ -0,0 +1,148 @@
+---
+id: getting-started
+title: Getting started
+description: Getting started
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - testing
+  - e2e
+  - getting-started
+sidebar_position: 2
+---
+
+import ScaffoldPluginE2InstallNPM from '@snippets/plugin-e2e-install.npm.md';
+import ScaffoldPluginE2EInstallPNPM from '@snippets/plugin-e2e-install.pnpm.md';
+import ScaffoldPluginE2EInstallYarn from '@snippets/plugin-e2e-install.yarn.md';
+import ScaffoldPluginE2EStartGrafanaNPM from '@snippets/plugin-e2e-start-grafana.npm.md';
+import ScaffoldPluginE2EStartGrafanaPNPM from '@snippets/plugin-e2e-start-grafana.pnpm.md';
+import ScaffoldPluginE2EStartGrafanaYarn from '@snippets/plugin-e2e-start-grafana.yarn.md';
+import ScaffoldPluginE2ERunTestsNPM from '@snippets/plugin-e2e-run-tests.npm.md';
+import ScaffoldPluginE2ERunTestsPNPM from '@snippets/plugin-e2e-run-tests.pnpm.md';
+import ScaffoldPluginE2ERunTestsYarn from '@snippets/plugin-e2e-run-tests.yarn.md';
+import ScaffoldPluginE2EDSWorkflowNPM from '@snippets/plugin-e2e-ds-workflow.npm.md';
+import ScaffoldPluginE2EDSWorkflowYarn from '@snippets/plugin-e2e-ds-workflow.yarn.md';
+import ScaffoldPluginE2EDSWorkflowPNPM from '@snippets/plugin-e2e-ds-workflow.pnpm.md';
+import ScaffoldPluginE2EProjectsConfigNPM from '@snippets/plugin-e2e-auth-project.npm.md';
+import ScaffoldPluginE2EProjectsConfigYarn from '@snippets/plugin-e2e-auth-project.yarn.md';
+import ScaffoldPluginE2EProjectsConfigPNPM from '@snippets/plugin-e2e-auth-project.pnpm.md';
+
+# Getting started
+
+This article will guide through how to install and configure `@grafana/plugin-e2e`, write tests and setup a basic Github workflow that can run your e2e tests targeting multiple versions of Grafana.
+
+## Prerequisites
+
+- You need to have a Grafana plugin [development environment](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment)
```

Comment:
```suggestion
- You need to have a suitable Grafana development environment where you can run your plugin, we recommend using the [docker development environment](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment) included with `create-plugin`
```

Reply:
This is not entirely true - a Grafana development environment is not required. 

What about something like this: 

- You need to have a Grafana plugin [development environment](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment)
- Grafana running on your local machine. We recommend using docker development environment included in plugin development environment, but you may also build Grafana from [source](https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md).